examples/private_set_intersection/module/rust/src/tests.rs

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use crate::proto::private_set_intersection::{GetIntersectionResponse, SubmitSetRequest};
+use assert_matches::assert_matches;
+use oak::grpc;
+use protobuf::well_known_types::Empty;
+use std::collections::HashSet;
+use std::iter::FromIterator;
+
+const MODULE_CONFIG_NAME: &str = "private_set_intersection";
+
+#[test]
+fn test_set_intersection() {
+    simple_logger::init().unwrap();
+
+    let (runtime, entry_channel) = oak_tests::run_single_module_default(MODULE_CONFIG_NAME)
+        .expect("Unable to configure runtime with test wasm!");
+
+    let mut req = SubmitSetRequest::new();
+    req.values.push("a".to_string());
+    req.values.push("b".to_string());
+    req.values.push("c".to_string());
+    let result: grpc::Result<Empty> = oak_tests::grpc_request(
+        &entry_channel,
+        "/oak.examples.private_set_intersection.PrivateSetIntersection/SubmitSet",
+        req,
+    );
+    assert_matches!(result, Ok(_));
+
+    let mut req = SubmitSetRequest::new();
+    req.values.push("b".to_string());
+    req.values.push("c".to_string());
+    req.values.push("d".to_string());
+    let result: grpc::Result<Empty> = oak_tests::grpc_request(
+        &entry_channel,
+        "/oak.examples.private_set_intersection.PrivateSetIntersection/SubmitSet",
+        req,
+    );
+    assert_matches!(result, Ok(_));
+
+    let req = Empty::new();
+    let result: grpc::Result<GetIntersectionResponse> = oak_tests::grpc_request(
+        &entry_channel,
+        "/oak.examples.private_set_intersection.PrivateSetIntersection/GetIntersection",
+        req,
+    );
+    assert_matches!(result, Ok(_));
+    let got = HashSet::<String>::from_iter(result.unwrap().values.to_vec());
+    let mut want = HashSet::new();
```

Comment:
https://docs.rs/maplit/1.0.2/maplit/ may be useful here.

Reply:
maplit looks helpful, but I'm loathe to add a dependency for the sake of saving two lines of code.  However,   looking further down in the `HashSet` docs there's a suggestion of `["b".to_string(), "c".to_string()].iter().cloned().collect()` which seems to fit the bill.